skills/roborazzi/references/top.md

```
@@ -0,0 +1,51 @@
+### Paparazzi and Roborazzi: A Comparison
+
+Paparazzi is a great tool for visualizing displays within the JVM. However, it's incompatible with
+Robolectric, which also mocks the Android framework.
+
+Roborazzi fills this gap. It integrates with Robolectric, allowing tests to run with Hilt and
+interact with components. Essentially, Roborazzi enhances Paparazzi's capabilities, providing a more
+efficient and reliable testing process by capturing screenshots with Robolectric.
+
+**Leveraging Roborazzi in Test Architecture: An Example**
+
+<img src="https://github.com/takahirom/roborazzi/assets/1386930/937a96a4-f637-4029-87e1-c1bb94abc8ae" width="320" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)